docs/deploy/aocl_fpga.md

```diff
@@ -0,0 +1,92 @@
+AOCL Backend Example
+====================
+
+TVM supports Intel FPGA SDK for OpenCL also known as AOCL.  Here is a tutorial for how to use TVM with AOCL.
+
+***Note***: This feature is still experimental.  We cannot use AOCL to deploy an end to end neural networks for now.  In addition, we only tested compilation for emulation mode of AOCL.
+
+We use two python scripts for this tutorial.
+
+- build.py - a script to synthesize FPGA bitstream.
+```
+import tvm
+
+tgt_host="llvm"
+tgt="aocl -device=de5net_a7 -mattr=emulator"
```

Comment:
I'd suggest "-device=s5_ref" for the tutorial.  It is the default device of aoc and available without installing additional BSP.

Reply:
Changed de5net_a7 to s5_ref.